views/templates/block/channelInfo.twig

```diff
@@ -1,47 +1,219 @@
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <span class="name">{{ upgradeInfo.version_name }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.version_num is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Version number:'|trans }}</label>
-                <span class="version">{{ upgradeInfo.version_num }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Version number:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <span class="version">{{ upgradeInfo.version_num }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.link is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'URL:'|trans }}</label>
-                <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'URL:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="url" href="{{ upgradeInfo.link }}">{{ upgradeInfo.link }}</a>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.md5 is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
-                <span class="md5">{{ upgradeInfo.md5 }}</span>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'MD5 hash:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <span class="md5">{{ upgradeInfo.md5 }}</span>
+                </div>
             </div>
         {% endif %}
         {% if upgradeInfo.changelog is not empty %}
-            <div style="clear:both;">
-                <label class="label-small">{{ 'Changelog:'|trans }}</label>
-                <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+            <div class="row">
+                <div class="col-xs-3 col-md-1">
+                    <label class="label-small">{{ 'Changelog:'|trans }}</label>
+                </div>
+                <div class="col-xs-9 col-md-11">
+                    <a class="changelog" href="{{ upgradeInfo.changelog }}">{{ 'see changelog'|trans }}</a>
+                </div>
             </div>
         {% endif %}
+        <div class="row">
+            <div class="col-xs-3 col-md-1">
+                <label class="label-small">{{ 'PHP compatibility:'|trans }}</label>
```

Comment:
I wonder since this table is built in HTML I guess the image is no longer needed? I didn't see where it's used actually, maybe it was used previously but not anymore?